### DISCUSSION TITLE
Make it possible to search for an Udi

### DISCUSSION THREAD

**Discussion by skttl** [umbraco/Umbraco-CMS#13406] [Features and ideas]
When content deployments using Deploy fails, you are often left with only a couple of Udis to start working out problem.

It would be nice if you could search for that udi and get the actual node (or whatever it represents). Currently, there is no results.

![image](https://user-images.githubusercontent.com/3726467/201903976-20e1c544-3399-42e0-908c-608e088d2d6d.png)

**Comment by lassefredslund** [comment]
Hi @skttl.

Sounds like a good idea. I'll set it to up-for-grabs.
